src/services/item/plugins/publication/validation/utils.ts

```diff
@@ -1,9 +1,34 @@
 import path from 'path';
 import striptags from 'striptags';
 
+import { ItemType, MimeTypes } from '@graasp/sdk';
+
 import { TMP_FOLDER } from '../../../../../utils/config';
+import { FolderItem, Item, isItemType } from '../../../entities/Item';
 
 export const stripHtml = (str?: string | null): string => (str ? striptags(str) : '');
 
 export const buildStoragePath = (itemId: string): string =>
   path.join(TMP_FOLDER, 'validations', itemId);
+
+export const assertItemIsFolder = (item: Item): FolderItem => {
```

Comment:
I don't know what the opinion of the team about how we're initialising functions. I feel like currently we're using both methods.
Personally, I'd use the `function assertItemIsFolder(item: Item): FolderItem` because it is hoisted and more readable. I'm only opening the discussion, I don't think it is necessary to make a decision about it.

Reply:
I've always used arrow functions because it was a recommendation from eslint. But I'm open to any.

Reply:
Well, I feel like the behavior of the 2 usages is very different, we should probably choose the right initialisation dependently of the use case.
To me, arrows functions are great for callbacks and offers a useful implicit return, but I'd leave the rest to the verbose one. 

I also saw we can initialise anonymous functions that way : 
```typescript
const foo = function() {}
```
Which doesn't make much sense to me, it would better suit to arrow function.

Reply:
I'm not sure that the behaviour is very different, except when using the `this` keyword. But if you have more information concerning the differences I'm really interested 😄.

Reply:
All what has been mentioned earlier already makes a big difference to me. There's still subtil differences that I don't think we're concerned: (`call()`, `apply()`, `bind()`, `arguments array` ) 

Otherwise, we're having difficulties to profile our code because most of our functions are anonymous

Reply:
Yes, I agree that there are differences, what I mean is that in our use, I haven't seen any limits to one compared to the other, except in the profiler because of the anonymous functions.

Again, I don't have any preference, so it's okay to me if we decide to always use functions, except when we have to use arrow function for some reason.

Reply:
I'll resolve this conversation, but we can maybe discuss more about that if needed in a new issue ?